README.md

```diff
@@ -246,6 +246,22 @@ req.checkBody({
 });
 ```
 
+You can also define location against to validate specific schema validator by adding `in` parameter as fallowing:
+
+```javascript
+req.checkBody({
+ 'email': {
+    in: 'query',
+    notEmpty: true,
+    isEmail: {
+      errorMessage: 'Invalid Email'
+    }
+  }
+```
+
+This will determine location for the specific validator no matter which method you use to call validator. If you use `in: 'query'` then checkQuery() will be called even if you called you schema as checkParams(). Currently supported location here are `'body', 'params', 'query'`.
+If you provide not supported location parameter then validation process for current parameter will be skipped.
```

Comment:
Change to:
If you provide a location parameter that is not supported, the validation process for current parameter will be skipped.